src/cortxfs/cortxfs_fops.c

```diff
@@ -184,21 +184,22 @@ static inline ssize_t __cfs_write(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
 
 	RC_WRAP_LABEL(rc, out, cfs_access_check, cred, stat, CFS_ACCESS_WRITE);
 
-	ssize_t bs = dstore_get_bsize(dstore, &oid);
-	if (bs < 0) {
-		rc = bs;
+	if (stat->st_blksize < 0) {
```

Comment:
Do we really needs this check? I assume that if FS got created means we have some valid bsize and hence IMHO this check is meaningless.